rundeckapp/grails-app/views/menu/home.gsp

```diff
@@ -25,391 +25,446 @@
 <%@ page import="grails.converters.JSON; org.rundeck.core.auth.AuthConstants" contentType="text/html;charset=UTF-8" %>
 <html>
 <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
-
-    <meta name="layout" content="base"/>
-    <meta name="tabpage" content="home"/>
-    <title><g:appTitle/></title>
-    <g:set var="uiType" value="${params.nextUi?'next':params.legacyUi?'legacy':'current'}"/>
-    <g:if test="${!projectNames}">
-        <g:embedJSON data="${[projectNames:[],projectNamesTotal:-1]}" id="projectNamesData"/>
-    </g:if>
-    <g:elseif test="${projectNames && projectNames.size()<50}">
-        <g:embedJSON data="${[projectNames:projectNames,projectNamesTotal:projectNames.size()]}" id="projectNamesData"/>
-    </g:elseif>
-    <g:elseif test="${projectNames}">
-        <g:embedJSON data="${[projectNames:projectNames[0..49],projectNamesTotal:projectNames.size()]}" id="projectNamesData"/>
-    </g:elseif>
-    <g:embedJSON data="${[loaded:statsLoaded,execCount:execCount,totalFailedCount:totalFailedCount,recentUsers:recentUsers,recentProjects:recentProjects]}" id="statsData"/>
-
-    <g:embedJSON data="${[
-            detailBatchMax        : params.getInt('detailBatchMax')?:cfg.getInteger(config: 'gui.home.projectList.detailBatchMax', default: 15),
-            summaryRefresh        : cfg.getBoolean(config: 'gui.home.projectList.summaryRefresh', default: true),
-            refreshDelay          : cfg.getInteger(config: 'gui.home.projectList.summaryRefreshDelay', default: 30000),
-            detailBatchDelay      : params.getInt('detailBatchDelay')?:cfg.getInteger(config: 'gui.home.projectList.detailBatchDelay', default: 1000).toInteger(),
-            pagingEnabled         : params.getBoolean('pagingEnabled',cfg.getBoolean(config: 'gui.home.projectList.pagingEnabled',default: true)),
-            pagingMax             : params.getInt('pagingMax')?:cfg.getInteger(config: 'gui.home.projectList.pagingMax', default: 30),
-    ]}" id="homeDataPagingParams"/>
-
-    <!-- VUE JS REQUIREMENTS -->
-    <asset:javascript src="static/pages/home.js" defer="defer"/>
-    <asset:javascript src="static/components/ko-paginator.js"/>
-    <!-- /VUE JS REQUIREMENTS -->
-
-    <asset:javascript src="menu/home.js"/>
-
-    <!-- VUE CSS MODULES -->
-    <asset:stylesheet href="static/css/components/version-notification.css"/>
-    <!-- /VUE CSS MODULES -->
-    <asset:javascript src="static/pages/login.js"/>
-    <style type="text/css">
-    .project_list_item_link{
-        display:inline-block;
+	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
```

Comment:
are the changes in this file, just to revert it to previous state before any vue conversion?

Reply:
Yes, I had previously added nextUi test in this file to decide when to render the vue component for the header, so removed that logic as well since it all lives inside a vue page.